aspnetcore/host-and-deploy/blazor/index.md

```diff
@@ -38,6 +38,45 @@ A Blazor client-side app is published to the */bin/Release/{TARGET FRAMEWORK}/pu
 
+## App base path
+
+The *app base path* is the app's root URL path. Consider the following main app and Blazor app:
+
+* The main app is called `MyApp`:
+  * The app physically resides at *d:\\MyApp*.
+  * Requests are received at `https://www.contoso.com/{MYAPP RESOURCE}`.
+* A Blazor app called `CoolApp` is a sub-app of `MyApp`:
+  * The sub-app physically resides at *d:\\MyApp\\CoolApp*.
+  * Requests are received at `https://www.contoso.com/CoolApp/{COOLAPP RESOURCE}`.
+
+Without specifying additional configuration for `CoolApp`, the sub-app in this scenario has no knowledge of where it resides on the server. For example, the app can't construct correct relative URLs to its resources without knowing that it resides at the relative URL path `/CoolApp/`.
+
+To provide configuration for the Blazor app's base path of `https://www.contoso.com/CoolApp/`, the `<base>` tag's `href` attribute is set to the relative root path in the *wwwroot/index.html* file:
+
+```html
+<base href="/CoolApp/">
+```
+
+By providing the relative URL path, a component that isn't in the root directory can construct URLs relative to the app's root path. Components at different levels of the directory structure can build links to other resources at locations throughout the app. The app base path is also used to intercept hyperlink clicks where the `href` target of the link is within the app base path URI space&mdash;the Blazor router handles the internal navigation.
+
+In many hosting scenarios, the relative URL path to the app is the root of the app. In these cases, the app's relative URL base path is a forward slash (`<base href="/" />`), which is the default configuration for a Blazor app. In other hosting scenarios, such as GitHub Pages and IIS sub-apps, the app base path must be set to the server's relative URL path to the app.
+
+To set the app's base path, update the `<base>` tag within the `<head>` tag elements of the *wwwroot/index.html* file. Set the `href` attribute value to `/{RELATIVE URL PATH}/` (the trailing slash is required), where `{RELATIVE URL PATH}` is the app's full relative URL path.
+
+For an app with a non-root relative URL path (for example, `<base href="/CoolApp/">`), the app fails to find its resources *when run locally*. To overcome this problem during local development and testing, you can supply a *path base* argument that matches the `href` value of the `<base>` tag at runtime. To pass the path base argument when running the app locally, execute the `dotnet run` command from the app's directory with the `--pathbase` option:
+
+```console
+dotnet run --pathbase=/{RELATIVE URL PATH (no trailing slash)}
+```
+
+For an app with a relative URL path of `/CoolApp/` (`<base href="/CoolApp/">`), the command is:
+
+```console
+dotnet run --pathbase=/CoolApp
+```
+
+The app responds locally at `http://localhost:port/CoolApp`.
+
 ## Deployment
 
 For deployment guidance, see the following topics:
```

Comment:
The "Blazor serverless hosting with Azure Storage" section below seems like it belongs in the Blazor client-side host and deploy topic as it is specific to Blazor WebAssembly apps.